src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -237,22 +237,11 @@ public bool IsAllProjectsNominated()
             // for VS14, always return true since nominations don't apply there.
             return true;
 #else
-            var netCoreProjects = GetNuGetProjects().OfType<NetCorePackageReferenceProject>().ToList();
-
-            foreach (var project in netCoreProjects)
-            {
-                // check if this .Net core project is nominated or not.
-                DependencyGraphSpec projectRestoreInfo;
-                if (!_projectSystemCache.TryGetProjectRestoreInfo(project.MSBuildProjectPath, out projectRestoreInfo) ||
-                    projectRestoreInfo == null)
-                {
-                    // there are projects still to be nominated.
-                    return false;
-                }
-            }
-
-            // return true if all the net core projects have been nominated.
-            return true;
+            // return true if all .Net core projects are nominated, else false.
+            DependencyGraphSpec projectRestoreInfo;
+            return GetNuGetProjects().OfType<NetCorePackageReferenceProject>().
```

Comment:
hmm, yeah. in either case I think we should get rid of ToList() to avoid iterating twice.